simplivity/resources/certificates.py

```diff
@@ -0,0 +1,52 @@
+###
+# (C) Copyright [2020] Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##
+from simplivity.resources.resource import ResourceBase
+
+"""Implements features available for certificate resource."""
+URL = '/security/certificates'
+DATA_FIELD = 'certificates'
+
+
+class Certificates(ResourceBase):
+    """Implements features for SimpliVity Certificates resources."""
+
+    def __init__(self, connection):
+        """Initialize Certificates class."""
+        super(Certificates, self).__init__(connection)
+
+    def get_all(self):
+        """Get all SSL certificates from the HPE SimpliVity trust store"""
+        return self._client.get_all(URL, members_field=DATA_FIELD)
+
+    def get_by_data(self, data):
+        """Gets Certificate object from data.
+
+        Args:
+            data: Certificate data
+
+        Returns:
+            object: Certificate object.
+        """
+        return Certificate(self._connection, self._client, data)
+
+
+class Certificate(object):
+    """Implements features available for single Certificate resource."""
+
```

Comment:
add OBJECT_TYPE class variable to make it align with other resource classes

Reply:
Hi @sijeesh  ,
Whenever we will implement the refresh method in the class we can add the  OBJECT_TYPE variable. However if we add this variable now it will be an unused in the class.

Let me know if we want to add the variable.